tracer/src/Datadog.Trace/RemoteConfigurationManagement/ProductConfigChangedEventArgs.cs

```diff
@@ -0,0 +1,22 @@
+// <copyright file="ProductConfigChangedEventArgs.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+
+namespace Datadog.Trace.RemoteConfigurationManagement;
```

Comment:
As per the comment we received on the the open-sourcing PR, we should use block-scope namespace declarations for now, for consistency with the rest of the codebase.

Reply:
Is that we said? As I understood it, we were using file-scoped namespaces on all _new_ files 🤔  I mean... that's what I've been doing for months anyway 😬

Reply:
No, that's what @tonyredondo said :) I'm good with either and don't have a strong opinion either way, but could be good to just make a decision and then use the Resharper command to change everything all at once for consistency, and then possibly enforce the decision it at compile time (just to reduce future friction)?

Reply:
> but could be good to just make a decision and then use the Resharper command to change everything all at once for consistency, and then possibly enforce the decision it at compile time (just to reduce future friction)

💯 

The _downside_ to this is that it affects every single file in the project, which means 100% merge conflicts, and it renders git blame useless (as every line was last touched by that commit!). Hence the currently slow conversion to file-scope namespaces as an intermediate approach..

Luckily, the git-blame issue can be resolved with a _.git-blame-ignore-revs_ file. I've been tracking GitHub's implementation for a while, and looks like [it finally made it in](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view) which fixes the blame concerns 🙂 

We still have the merge conflict issue though which is non-trivial... We'll just have to bite the bullet at some point when people are happy. I'll bring it up at next eng meeting!